docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -53,8 +53,168 @@ Intel® Core™ i3-8121U Processor
 
 * All [11th Generation Intel® Core™ Processors (formerly codenamed Tiger Lake)](https://ark.intel.com/content/www/us/en/ark/products/codename/88759/tiger-lake.html).
 
+* All [12th Generation Intel® Core™ Processors (formerly codenamed Alder Lake)](https://ark.intel.com/content/www/us/en/ark/products/codename/147470/products-formerly-alder-lake.html).
+
 > **NOTE**: On platforms where Intel® GNA is not enabled in the BIOS, the driver cannot be installed, so the GNA plugin uses the software emulation mode only.
 
+### Intel® GNA Generational Differences
+
+The first and second versions of Intel® GNA found in 10th and 11th generation Intel® Core™ Processors may be considered to be functionally equivalent.  Intel® GNA 2.0 provided performance improvement with respect to Intel® GNA 1.0.  Starting with 12th Generation Intel® Core™ Processors (formerly codenamed Alder Lake), support for Intel® GNA 3.0 features is being added.
+
+In the rest of this documentation, "GNA 2.0" term will be used for GNA HW delivered with  Intel® GNA on 10th and 11th generation Intel® Core™ Processors, and "GNA 3.0" term will be used for GNA HW delivered with  Intel® GNA on 12th generation Intel® Core™ Processors.
```

Comment:
Should say:  the term "GNA 2.0" refers to Intel(R) GNA hardware delivered on 10th and 11th generation Intel(R) Core(TM) processors, and the term "GNA 3.0" will be used to refer to GNA hardware delivered on 12th generation Intel(R) Core(TM) processors.